### PR TITLE
LPS-182206 Notification Template screen broken on large displays

### PR DESCRIPTION
**Pull request by Valmir-unicap**
Related issue: https://issues.liferay.com/browse/LPS-182206

To reproduce this bug, change your monitor resolution to (2560 x 1440).

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @cgmonte, 
Could you review this PR please? :smile:

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.scss [hunk 1]
```diff
@@ -35,6 +35,7 @@
 }
 
 @include responsiveCards(3840px, 40rem);
```
Comment (cgmonte) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.scss:37]: This line creates a range between 2160px and 3840px, which includes 2560px, so your change in the next line is not necessary.

**Review by cgmonte — COMMENTED** [review]
Hey @Valmir-unicap Thanks for sending this PR! To be fair, I couldn't even reproduce this bug and your change creates a new width range without changing the margin it applies. Please, let me know when you're at the office, because I would like to see this bug reproduced in your machine.

**Comment by cgmonte** [comment]
Closing this because the fix does nothing and also because there is nothing to fix since several people tried to reproduce the bug and were not able to do it. The ticket went back to `open` and QA will needs to verify it again. There must be other conditions to trigger the bug besides screen resolution.

**Pull request closed without merging**